jolt-core/src/jolt/vm/timestamp_range_check.rs

```diff
@@ -284,11 +290,11 @@ where
         polynomials: &RangeCheckPolynomials<F, C>,
         gamma: &F,
         tau: &F,
-    ) -> (Vec<DensePolynomial<F>>, Vec<DensePolynomial<F>>) {
+    ) -> (Vec<Vec<F>>, ()) {
```

Comment:
nit: Add doc comment that InitFinal leaves are batched into ReadWrite leaves.

Reply:
done